cmd/juju/commands/main.go

```diff
@@ -314,6 +314,7 @@ func registerCommands(r commandRegistry, ctx *cmd.Context) {
 	r.Register(newUpgradeJujuCommand(nil, nil))
 	r.Register(application.NewUpgradeCharmCommand())
 	r.Register(application.NewUpdateSeriesCommand())
+	r.Register(application.NewSetSeriesCommand())
```

Comment:
Does this need to be behind `JUJU_DEV_FEATURE_FLAGS=upgrade-series"`?

Reply:
I don't think so. This command existed, is tested, etc. It's just a new name for it. No sense putting it behind the flag atm.